docker/base/conf.d/docker.conf

```diff
@@ -0,0 +1,7 @@
+## When running Nuxeo in a Linux container, the JVM will automatically detect the cgroup memory limit with the
+## UseContainerSupport option, enabled by default.
+## By default, we set the minimum and maximum heap size to 50% of the cgroup memory limit.
+## This way:
+## - The minimum and maximum are equal, which is recommended in production, as we used to recommend Xms == Xmx.
```

Comment:
By default, we set the heap size to a fixed size equals to 50% of the cgroup memory limit.
This is achieved by using the same percentage for initialRAM as MaxRAM resulting in equals Xms and Xmx which is recommended in production.